gatsby/src/docs/accounts/sso/azure-sso.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Azure Active Directory SSO'
+sidebar_order: 1
+---
+
+## Installation
+
+1. Login to the [Azure portal](https://portal.azure.com/).
+
+2. (You can skip to (3) if you don't need to create a permission group for Sentry.) Under 
+Manage Azure Active Directory (the shield picture), click "View", which takes
+you to the Organization Overview page. On the left hand navigation, click "Groups". Create a 
+new group, add an owner, and members to the group. Set "Group type" to Office 365. For more 
+details about group creation, see the
+[Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+3. On the left hand navigation, click "Groups". Create a new group, add an owner, and
+members to the group. Set "Group type" to Office 365. For more details about group creation, see the
+[Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+4. Return to the Overview page. On the left hand navigation, click "Enterprise Applications".
+Then, select "Non-gallery application" and create one for Sentry. Once created, you'll
+be directed to "Sentry - Overview."
+
+5. Click on `1. Assign users and groups`, and assign yourself and the group you've created 
+to the Sentry app.
+
+6. Click on `2. Setup Single Sign-on`
+
+7. For Section (1) labeled as `Basic SAML2 Configuration`, enter the following data 
+in each line.
+
+    
+    Identifier (entity ID): https://sentry.io/saml/metadata/sentry/
```

Comment:
should these be formatted as code? Example - Identifier (entity ID): `https://sentry.io/saml/metadata/sentry/`?